examples/stream_queues/receive_stream.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+
+var amqp = require('amqplib');
```

Comment:
can you please rewrite to use modern style with `const`/`let` and `async/await`?